requirements/base.in

```diff
@@ -2,8 +2,7 @@ celery==4.4.2
 django-debug-toolbar==3.1.1
 django-filter==2.4.0
 flower==0.9.5
-psycopg2==2.8.6
-requests-mock==1.8.0
+psycopg2-binary==2.8.6
```

Comment:
usage of binary is discouraged on production systems

Reply:
https://pypi.org/project/psycopg2-binary/
"The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources."